### DISCUSSION TITLE
How to use cpp in pulsar? Windows User.

### DISCUSSION THREAD

**Discussion by Its-anand** [pulsar-edit/.github#200] [Community Package Help] [answered]

**Comment by confused-Techie** [comment, marked as answer]
I'm not a C++ developer, so might be missing why it's necessary to set the Pulsar Path anywhere for this to work.

But on Windows if you did a User (the default) install, that path should look like: `C:\Users\YOUR_USER_NAME_HERE\AppData\Local\Programs\pulsar` otherwise: `%UserProfile%\AppData\Local\programs\pulsar`

EDIT:

I'll add, if you're a Windows user who did not use the default installation directory, but you can't remember where you installed Pulsar, an easy way to locate your installation folder would be by opening the DevTools within Pulsar then running `atom.applicationDelegate.getWindowLoadSettings().resourcePath`, this will return something like: `"C:\Users\USER_NAME\AppData\Local\Programs\pulsar\resources\app.asar"`, then ignoring the `resources/app.asar` on the end is the top level directory of Pulsar's install location, no matter how you installed it on Windows. (Except if running the portable version)
